src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -312,7 +312,7 @@ public final class TokenTypes {
      *
      * <p>For example:</p>
      * <pre>
-     * final int PI = 3.14;
+     * final double PI = 3.14;
```

Comment:
`--LITERAL_INT`

This line below should be changed .

Reply:
Oh right. Thanks.